assets/containers/Cpp/README.md

```diff
@@ -0,0 +1,54 @@
+# Azure IoTEdge & C# - .NET Core 3.1
+
+## Summary
+
+*Develop Azure IoTedge Solutions in C, the device C SDK, and related extensions and dependencies.*
+
+| Metadata | Value |  
+|----------|-------|
+| *Contributors* | The Azure IoT Device Experience Team, @mhshami01 |
+| *Categories* | Devices, Azure |
+| *Definition type* | Dockerfile |
+| *Works in Codespaces* | Yes |
+| *Container host OS support* | Linux, macOS, Windows |
+| *Container OS* | Debian |
+| *Languages, platforms* | C Device SDK | C/C++ |
+
+## Using this definition
+
+This definition requires an Azure subscription to use. You can create a [free account here](https://azure.microsoft.com/en-us/free/serverless/). Once you have an Azure account, you can learn more about [Azure IoT Edge](https://docs.microsoft.com/en-us/azure/iot-edge/about-iot-edge?view=iotedge-2020-11) and how to [Develop Custom Modules with Visual Studio Code](https://docs.microsoft.com/en-us/azure/iot-edge/how-to-vs-code-develop-module?view=iotedge-2020-11). Once you have an IoT Edge device setup, follow these steps:
+
+1. If this is your first time using a development container, please see getting started information on [setting up](https://aka.ms/vscode-remote/containers/getting-started) Remote-Containers or [creating a codespace](https://aka.ms/ghcs-open-codespace) using GitHub Codespaces.
+
+2. Start VS Code and open your project folder or connect to a codespace.
+
+3. Press <kbd>F1</kbd> select and **Add Development Container Configuration Files...** command for **Remote-Containers** or **Codespaces**. 
+
+   > **Note:** If needed, you can drag-and-drop the `.devcontainer` folder from this sub-folder in a locally cloned copy of this repository into the VS Code file explorer instead of using the command.
+
```

Comment:
i don't think that developers should clone the repo in order to use this new feature, all should be wrapped up in the vsix so they won't have the ability to drag and drop the folder locally